index.js

```diff
@@ -60,6 +60,16 @@ function serveSinglePageApp(request) {
   } catch {}
 
   if (request.url.endsWith('.html') && reactRouting) {
+    const url = new URL(request.url);
```

Comment:
This snippet looks good, but Im not sure it will work because of `request.url.endsWith('.html')`

Does this need to be outside of the condition? Otherwise, it will only match on paths like `/some/thing.html`

Reply:
🤔 this was there previously... only time this will trigger is if request.url specifically ends with .html. usually the request does not... eg. 

https://cloudflareworkers.com/#cb6e4f4039dfd1e65fa15a9d1634e689:http://about:blank/
https://cloudflareworkers.com/#cb6e4f4039dfd1e65fa15a9d1634e689:http://about:blank/index.html